client/my-sites/plan-features-2023-grid/actions.tsx

```diff
@@ -317,6 +319,25 @@ const PlanFeaturesActionsButton: React.FC< PlanFeaturesActionsButtonProps > = (
 					  } ) }
 			</Button>
 		);
+	} else if ( isWooExpressPlusPlan ) {
+		const translateComponents = {
+			ExternalLink: (
```

Comment:
Nit: given that we're only using this once, I would prefer this to be inline so it comes after the text. It looks like some of the other places where this is pushed into a `const` are because the values are used in two places.